app/src/main/java/de/westnordost/streetcomplete/quests/crossing_markings/AddCrossingMarkings.java

```diff
@@ -0,0 +1,44 @@
+package de.westnordost.streetcomplete.quests.crossing_markings;
```

Comment:
Please disable for now - we will want to add as ways, hopefully under a different schema.